### ISSUE TITLE
Center topic references on wider screens

### ISSUE THREAD

**Issue by calvinf** [calvinf/wellversed-ios#15]
As a user, it's odd to have so much extra padding on the topic scene's cards.  Dynamically analyze the width and orientation and based on that set padding to get the references centered.

**Comment by calvinf** [comment]
Addressed by d8e201b982e1791ead284a816bcb3c80c6d758fc.

Remaining tasks:
* Update code to make it work against screen rotation
* See if we can unit test this by mocking results of Dimensions

**Comment by calvinf** [comment]
See this repo for a way to track orientation changes:
* https://github.com/yamill/react-native-orientation
* https://github.com/yamill/react-native-orientation/tree/master/RCTOrientation

**Comment by calvinf** [comment]
I tried a couple orientation solutions, but those seem messy, require adding extra orientation-related bridges and npm modules, and both the modules I looked at don't receive much maintenance.

Looking at react native documentation more, I believe I can use the onLayout property of the relevant `View` in the TMS scene.  Based on layout changes and changes to the width, I can recalculate the `columnWidth` value.

I began work with this commit: aea2f791be8243a9820548a7f040a916f69030fd
